glide.yaml

```diff
@@ -32,14 +33,15 @@ import:
 - package: github.com/uber-common/bark
   version: fix-logrus-8841a0f
 - package: github.com/uber/cherami-client-go
-  version: =1.23.0
+  version: =2.3.0
   subpackages:
   - client/cherami
   - common
   - common/backoff
   - common/websocket
   - stream
 - package: github.com/uber/cherami-thrift
+  version: v1.24.0-rc0
```

Comment:
I would remove the -rc0, to be consistent with the other repos. We can make this a pre-release anyway.